index.html

```diff
@@ -81,7 +83,7 @@ <h2 class="jamilababtext">THIS SITE IS FREE TO USE! CLICK HERE TO SEND A TIP ❤
       <button class="close-modal">&times;</button>
       <h1>As-salamu alaykum 👋</h1>
       <p>
-        The calculated times displayed above are accurate to the nearest minute. In your case, this means if you pray or make dua at least 1 minute after <span class="userthird"></span>, this means you have definitely prayed or made dua in the last third of the night. Click <a href="https://islamqa.info/en/answers/291824/the-virtue-of-offering-supplication-in-the-last-third-of-the-night">here</a> to understand why the last third of the night is a special time for Muslims. As for the midnight value given to you (<span class="usermidnight"></span>), this is for those who believe the ‘Isha prayer must be performed before midnight. This is also accurate to the nearest minute, so this could mean midnight is actually 1 minute earlier than the value given to you. I must remind you that the calculator depends on you giving accurate times for Maghrib and Fajr, so you must ensure they are correct. Report issues to me on <a href="https://instagram.com/proateen">Instagram</a>. The code for this site is on <a href="https://github.com/proateen">GitHub</a>. The code for this calculator has recently been improved and it now supports ALL countries.
+        The calculated times displayed above are accurate to the nearest minute. In your case, this means if you pray or make dua at least 1 minute after <span class="userthird"></span>, this means you have definitely prayed or made dua in the last third of the night. Click <a href="https://islamqa.info/en/answers/291824/the-virtue-of-offering-supplication-in-the-last-third-of-the-night">here</a> to understand why the last third of the night is a special time for Muslims. As for the midnight value given to you (<span class="usermidnight"></span>), this is for those who believe the ‘Isha prayer must be performed before midnight. This is also accurate to the nearest minute, so this could mean midnight is actually 1 minute earlier than the value given to you. As for the first third of the nigth value(<span class="userfirstthird"></span>), this gives to you approximate last time of Isha that are mentioned in some <a href="https://sunnah.com/abudawud:395">Hadiths</a>. I must remind you that the calculator depends on you giving accurate times for Maghrib and Fajr, so you must ensure they are correct. Report issues to me on <a href="https://instagram.com/proateen">Instagram</a>. The code for this site is on <a href="https://github.com/proateen">GitHub</a>. The code for this calculator has recently been improved and it now supports ALL countries.
```

Comment:
Can you please edit your addition:

As for the first third of the nigth value(<span class="userfirstthird"></span>), this gives to you approximate last time of Isha that are mentioned in some <a href="https://sunnah.com/abudawud:395">Hadiths</a>. 

to this:

As for the first third of the night value (<span class="userfirstthird"></span>), if you pray 'Isha at least 1 minute after (<span class="userfirstthird"></span>), you would have prayed after the first third of the night. To understand why this time is important, please watch <a href="https://www.youtube.com/watch?v=CtDucRQfACU">this</a> video until the end. 

@mehmet-kadioglu